example/main.go

```diff
@@ -39,18 +39,24 @@ import (
 var log = logf.Log.WithName("example-controller")
 
 func main() {
+	var enableInstaller bool
+	flag.BoolVar(&enableInstaller, "enable-installer", false,
+		"enable the installer in the webhook server, so it will install webhook related resources during bootstrapping")
+
```

Comment:
shall we rename the flag to reflect the usage ? for ex. `--install-webhook-config` ?